gax-grpc/version.txt

```diff
@@ -1 +1 @@
-0.18.1-SNAPSHOT
+0.19.0
```

Comment:
Seems like things could break as a result of the auth change, but I'll follow suit. Done.